htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/loader/decorator/HtmlElementLocatorFactory.java

```diff
@@ -40,6 +40,14 @@ public ElementLocator createLocator(Field field) {
      * @param clazz Class for which locator will be created.
      */
     public ElementLocator createLocator(Class clazz) {
-        return new AjaxElementLocator(searchContext, new HtmlElementClassAnnotationsHandler(clazz));
+        return new AjaxElementLocator(searchContext, getTimeOutInSeconds(), new HtmlElementClassAnnotationsHandler(clazz));
+    }
+
+    public static int getTimeOutInSeconds() {
+        try {
+            return Integer.valueOf(System.getProperty("webdriver.timeouts.implicitlywait"));
```

Comment:
It would be great if this can be overridden on per-element level using a `@Timeout` annotation for example.